File: temp.py
import os
files = ['dashboard.html', 'login.html', 'checkout.html', 'pos.html', 'invoice.html']
footer = '<div style="text-align:center; padding: 20px; font-size: 0.8rem; color: #94a3b8; width: 100%; position: relative; z-index: 100;">Developed by ASG</div>\n</body>'
for f in files:
    if os.path.exists(f):
        with open(f, 'r', encoding='utf-8') as file:
            content = file.read()
        if 'Developed by ASG' not in content:
            content = content.replace('</body>', footer)
            with open(f, 'w', encoding='utf-8') as file:
                file.write(content)
            print(f'Added footer to {f}')

File: temp.js
const fs = require('fs');
const files = ['dashboard.html', 'login.html', 'checkout.html', 'pos.html', 'invoice.html'];
const footer = '<div style="text-align:center; padding: 20px; font-size: 0.8rem; color: #94a3b8; width: 100%; position: relative; z-index: 100;">Developed by ASG</div>\n</body>';

files.forEach(f => {
  if (fs.existsSync(f)) {
    let content = fs.readFileSync(f, 'utf8');
    if (!content.includes('Developed by ASG')) {
      content = content.replace('</body>', footer);
      fs.writeFileSync(f, content);
      console.log('Added footer to ' + f);
    }
  }
});
